### PR TITLE
Deprecate NxDropdown variant prop - RSC-1431

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-1431

### CODE REVIEW

File: gallery/src/components/NxDropdown/NxDropdownPage.tsx [hunk 1]
```diff
@@ -85,6 +85,11 @@ const NxDropdownPage = () =>
               <NxTable.Cell>
                 What type of button to render for the dropdown.
                 Defaults to <NxCode>"tertiary"</NxCode>
+
+                <NxWarningAlert>
+                  Deprecated: This prop should not be used.
+                  It produces button styling that does not look good.
```
Comment (avnvbhatta) [inline gallery/src/components/NxDropdown/NxDropdownPage.tsx:91]:
```suggestion
                  It produces button styling that does not comply with design standards.
```

File: gallery/src/components/NxDropdown/NxDropdownPage.tsx [hunk 2]
```diff
@@ -85,6 +85,11 @@ const NxDropdownPage = () =>
               <NxTable.Cell>
                 What type of button to render for the dropdown.
                 Defaults to <NxCode>"tertiary"</NxCode>
+
```
Comment (avnvbhatta) [inline gallery/src/components/NxDropdown/NxDropdownPage.tsx:88]:
```suggestion
```

File: gallery/src/components/NxDropdown/NxDropdownPage.tsx (hunk 1 above)
Comment (avnvbhatta) [reply gallery/src/components/NxDropdown/NxDropdownPage.tsx:91]: Let me know your thoughts on this. Might be a bit better than "does not look good".

Comment (entroform) [reply gallery/src/components/NxDropdown/NxDropdownPage.tsx:90]: >Let me know your thoughts on this. Might be a bit better than "does not look good".
sounds good to me!

**Pull request merged**